specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4275,22 +4289,24 @@
 		}],
 		"properties": {
 			"textExtractionAlgorithm": {
-				"type": "string",
-				"default": "printed",
+				"$ref": "#/definitions/TextExtractionAlgorithm",
 				"description": "A value indicating which algorithm to use. Default is printed."
 			},
 			"defaultLanguageCode": {
-				"type": "string",
-				"default": "en",
+				"$ref": "#/definitions/OcrSkillSupportedLanguageCodes",
 				"description": "A value indicating which language code. Default is en."
 			},
 			"detectOrientation": {
+			    "x-ms-client-name": "ShouldDetectOrientation",
 				"type": "boolean",
 				"default": false,
 				"description": "A value indicating to turn orientation detection on or not. Default is false."
 			}
 		},
-		"description": "Computer vision OCR."
+		"externalDocs": {
+			"url": "https://docs.microsoft.com/azure/search/cognitive-search-skill-ocr"
```

Comment:
These doc links are going to very helpful to customers. Thanks for putting them here!

Reply:
Thanks. :)